File: PClassic/2022s/Stubs/9/Biblioburro.py
import math

def efficientTrip(mountainList):
    return True


if __name__ == '__main__':
    tests = int(input())
    for test in range(tests):
        length = int(input())
        line = input().split()
        mountainList = []
        for e in line:
            mountainList.append(int(e))
        res = efficientTrip(mountainList)
        if res:
            print(1)
        else:
            print(0)

File: PClassic/2022s/Stubs/9/Biblioburro.cpp
#include <iostream>
#include <vector>

using namespace std;


bool efficientTrip(vector<long long> mountainList) {
    return true;
}


int main() {

    int t; 
    cin >> t;

    while (t--) {
        int n; 
        cin >> n;
        vector<long long> mountainList;
        mountainList.resize(n);
        for (int i = 0; i < n; i++)
            cin >> mountainList[i];

        bool res = efficientTrip(mountainList);
        if (res) {
            cout << 1 << endl;
        } else {
            cout << 0 << endl;
        }
    }
}